build-local.sh

```diff
@@ -1,2 +1,6 @@
 #!/bin/bash
-mvn -DskipTests clean package && docker build -t govukpay/publicauth:local .
+
+set -e
+
+mvn -DskipITs -DskipTests clean package
```

Comment:
I guess you added `skipITs` because we want to skip all the tests when building locally? Not sure why we skip the tests at all when building locally with `skipTests`, but I think it's fine to add `skipITs` for consistency as well or we need to remove both of them